content/blog/2022-02-25-react-v18-rc.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "React v18.0 Release Candidate: Upgrade Guide"
+author: [rickhanlonii]
+---
+
+Today we’re publishing the second release candidate for React 18. With this release, we’re encouraging users to install and test React 18 to [raise any issues](https://github.com/facebook/react/issues) for any problems you encounter in the upgrade before we publish the full stable release.
+
+As we shared at [React Conf](https://reactjs.org/blog/2021/12/17/react-conf-2021-recap.html), React 18 will include our new concurrent renderer and a gradual migration strategy for concurrent features. React 18 also includes a small number of breaking changes and features outside of concurrent rendering. In this post we will describe the changes to expect when upgrading to React 18.
+
+*Note for React Native users: React 18 will ship in React Native with the New React Native Architecture. For more information, see the [React Conf keynote here](https://www.youtube.com/watch?v=FZ0cG47msEk&t=1530s).*
+
+## Installing
+
+To install the latest React 18 RC, use the `@rc` tag:
+
+```bash
+npm install react@rc react-dom@rc
+```
+
+Or if you’re using yarn:
+
+```bash
+yarn add react@rc react-dom@rc
+```
+
+## New client root APIs
+
+When you first install React 18, you will see a warning in the console:
+
+![Screenshot of React error with text "Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17."](../images/blog/react-v18-rc/reactdom_warning.png)
+
+React 18 introduces a new root API which provides better ergonomics for managing roots. The new root API also enables the new concurrent renderer, which allows you to opt-into concurrent features.
+
+To upgrade `ReactDOM.render`:
+
+```js
+// Before
+const container = document.getElementById('app');
+ReactDOM.render(<App tab="home" />, container);
+unmountComponentAtNode(containter); // ...some time later
```

Comment:
We already have a section later for this. It's also not mirrored in the corresponding "After" section.

Reply:
Nice catch, I was originally going to roll the examples together.